### PR TITLE
doc(release): prepare release note for MBI 22.10.1

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Prepare release notes for MBI 22.10.1

## Target version

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [x] 22.10.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [ ] 23.04.x (next)

### CODE REVIEW

**Review by cg-tw — APPROVED** [review]

**Review by jdelpierre — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -108,6 +108,14 @@ Release date: `October 26, 2022`
 
 ## Centreon MBI
 
+### 22.10.1
+
+Release date: `February 9, 2023`
+
+#### Bug fixes
+
+- [ETL] Change the name of the scripts called by the ETL
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:117]:
```suggestion
- [ETL] Fixed the name of the scripts called by the ETL.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -109,6 +109,12 @@ Release date: `October 26, 2022`
 
 ## Centreon MBI
 
+### 22.10.1
+
+Release date: `February 9, 2023`
+
+- [ETL] Change the name of the scripts called by the ETL
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:116]:
```suggestion
- [ETL] Fixed the name of the scripts called by the ETL.
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md (hunk 1 above)
Comment (jdelpierre) [reply i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md]: Yes, it's ok

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md (hunk 2 above)
Comment (jdelpierre) [reply versioned_docs/version-22.10/releases/centreon-commercial-extensions.md]: Yes, it's ok

**Review by jdelpierre — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

**Pull request merged**